### PR TITLE
add test_validations

### PR DESCRIPTION
**Pull request by crgjr66**
- added validations for presence of name, age, and enjoys
- added validation on length of enjoys > 10
- added check for http status on the create method

### CODE REVIEW

File: spec/requests/cats_request_spec.rb [hunk 1]
```diff
@@ -81,4 +81,42 @@
 
   end
 
+  it "doesn't create a cat without a name" do
```
Comment (sjproctor) [inline spec/requests/cats_request_spec.rb:84]: Looks like you need validations for age and enjoys too!

**Review by sjproctor — CHANGES_REQUESTED** [review]
Looks like you are missing two request tests.

File: spec/requests/cats_request_spec.rb (hunk 1 above)
Comment (crgjr66) [reply spec/requests/cats_request_spec.rb:84]: added test validations for age and enjoys

**Review by sjproctor — APPROVED** [review]
Looks great!

**Pull request merged**